src/cba-list/cba-list.js

```diff
@@ -584,13 +598,21 @@ export class List extends HTMLElement
   _render()
   {
     this.container.dataset.subitems = this.hasSubtiems;
+    /**
+     * @param {ListItem|ListSubItem} item - list item.
+     */
     const createRow = (item) =>
     {
       const {text, selected, editable = false} = item;
       const classes = ["row"];
       if (selected)
         classes.push("highlight");
-      const row = html`<span class="${classes.join(" ")}" tabindex="${selected ? 0 : -1}" draggable="${this.drag}" contenteditable="${editable}" title="${text}">${text}</span>`;
+      let alertTmpl = "";
+      if (item.alert)
+      {
+        alertTmpl = html`<span class="alert ${item.alert.type}"><cba-tooltip text="${item.alert.text}" arrow="x">${infoIcon}</cba-tooltip></span>`;
+      }
+      const row = html`<span class="${classes.join(" ")}" draggable="${this.drag}" title="${text}" tabindex="${selected ? 0 : -1}"><span contenteditable="${editable}" class="text">${text}</span>${alertTmpl}</span>`;
```

Comment:
This needs to be revisited and tested better multiple issue noticed so far:
- Rename misses the focus.
- Bullet points are miss-aligned.

(Also when adjusting ensure that keyboard navigations still work)